crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -781,6 +781,7 @@ en:
     error:
       destroy_failed: 'Failed to destroy %{component}'
     crowbar:
+      disabled: 'This API is currently disabled in crowbar.yml config file (%{option} option).'
```

Comment:
should remove this text from the eperimental part in line 780, probably also remove the whole experimental section starting on line 779 as we have no longer text that is part of experimental.

Reply:
and probably use a more descriptive name for the key as `disabled` is really generic but it worked under the experimental key. Maybe `disallow_chef_restart_disabled`?

Reply:
I agree